object_manager.go

```diff
@@ -536,3 +537,26 @@ func (objMgr *ObjectManager) GetGridInfo() ([]Grid, error) {
 	err := objMgr.connector.GetObject(gridObj, "", &res)
 	return res, err
 }
+
+// Get the all networks
+func (objMgr *ObjectManager) GetNetworks(netview string, cidr string, ea EA) (*[]Network, error) {
```

Comment:
There is already a `GetNetwork()` method which performs essentially the same function, except that it returns only the first network. So, I suggest doing one of two things:
- Have only one `GetNetwork()` method that returns multiple networks
- If the reasoning is to prevent breaking API, then at least have `GetNetwork()` uses `GetNetworks()`, instead of duplicating the code.